tests/dispenser/TestFundCommand.test_fund_command_http_error.approved.txt

```diff
@@ -0,0 +1,2 @@
+DEBUG: Error processing dispenser API request: Limit exceeded
+ERROR: Error: Limit exceeded
```

Comment:
Should we make this message a bit more descriptive, like "Error: You exceeded the limit for the amount you can dispense; try again in x hours" or whatever